README.md

```diff
@@ -42,7 +42,7 @@ accounts, so this is what I used. I want to enable custom OAuth providers soon.
 Some of these system dependencies can be installed on a macOS development
 machine with the `bin/system` command using [Homebrew](https://brew.sh/).
 
-- Ruby 2.2.0
+- Ruby 2.5.1
```

Comment:
I know that there are some Ruby version overrides via `CUSTOM_RUBY_VERSION`, but the project setup is fairly locked into version `2.5.1`. Since this is a stand alone application, I'm assuming there's no reason to support lower versions anyhow?

Reply:
Agreed.